hathor/prometheus.py

```diff
@@ -26,6 +26,8 @@
 
 settings = HathorSettings()
 
+METRIC_PREFIX = 'hathor_core:'
```

Comment:
Should we use the same prefix added in the other PR with the cli command?

Reply:
Yes, I'll do this when the other one is merged

Reply:
I did a refactor in the merge: https://github.com/HathorNetwork/hathor-core/pull/467/commits/a16d9077d3fc33cddc97bcbfd0c3b15f7074802b